metacatalog_api/router/api/share_provider.py

```
@@ -1,3 +1,595 @@
+
+@share_router.post('/share/download/submit')
+async def submit_download(entry_id: int, request: Request):
+    """
+    Submit download request and return package
+    """
+    # Parse request body
+    try:
+        body = await request.json()
+        metadata_formats = body.get('metadata_formats', ['json', 'datacite'])
+        include_data = body.get('include_data', True)
+    except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)